### DISCUSSION TITLE
Cannot transfer issues

### DISCUSSION THREAD

**Discussion by DavidSagan** [community/community#67509] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When trying to transfer issues between two repos I am able to transfer some issues but other issues give an error. For example trying to use `gh`
```
> gh issue transfer https://github.com/bmad-sim/bmad-old/issues/156 bmad-sim/bmad-ecosystem
GraphQL: Something went wrong while executing your query. Please include `F2B4:9043:3E3CB1:7E4BA9:6508890B` when reporting this issue.
```
When I try transferring this issue via the browser, when I press the `Transfer issue` button I get an error message:

![Screenshot 2023-09-18 at 1 37 22 PM](https://github.com/community/community/assets/1026006/9a030b80-a031-44c2-84c3-c1354658a65c)

**Comment by DavidSagan** [comment]
I just tried again and now I was able to transfer the issue:
```
> gh issue transfer https://github.com/bmad-sim/bmad-old/issues/156 bmad-sim/bmad-ecosystem
https://github.com/bmad-sim/bmad-ecosystem/issues/162
```

I did not do anything that would have fixed the issue so I'm guessing this is an intermittent problem.

**Reply by iampremo** [reply]
I was getting this the other day while bulk transferring issues, it looked to be some sort of hidden rate limit as retying some time later would allow me to transfer again. After a bit of back and forth I was told they would investigate further..
